### ISSUE TITLE
User should be able to do a "scheduled send"

### ISSUE THREAD

**Issue by holmesworcester** [TryQuiet/quiet#2031]

**Comment by jgaylor** [comment]
@holmesworcester 

Gmail and Slack both support Drafts. Beyond obvious reasons, this allows scheduled messages to be "Cancelled" at which time they are reverted to Drafts.

1. If we support drafts like Google and Slack, then canceling converts your message to a draft.
2. If we don’t support drafts should users be able to cancel or delete a scheduled message?
3. What would happen if you cancel? 

Options:
A. Don't let user's take any action on scheduled messages
B. Offer a single option to delete the message entirely
C. Offer a cancel option which then reverts it to a draft.

Wdyt?

**Comment by holmesworcester** [comment]
I mean, minimal drafts are just "remember what I was typing in each channel when I switch away." This is definitely something we'll have.

How we display it is another question. I think C is safe.

**Comment by jgaylor** [comment]
Right, but if you have multiple scheduled messages and start cancelling
them, they will need to go into a list somewhere to access, vs just taking
you back to the channel where you were drafting the message. I think it
would need to be more than minimal because of multiple. I’ll go with the
Slack route unless I hear otherwise.

**Comment by jgaylor** [comment]
Already shared in Slack last week, just adding these to the issue: 

The latest [Loom](https://www.loom.com/share/51b293fe31944d688f1e7102b3540cc0) and [Figma](https://www.figma.com/file/WcMFt8ntHvNXammfEosbbZ/Scheduled-send?type=design&node-id=7%3A2510&mode=design&t=SXZTN76WV0MzH1FL-1).

**Comment by holmesworcester** [comment]
Noting that we've decided to change where we put messages that are scheduled to be sent. We should update this design to keep the issue fresh, and get feedback on it again.

**Comment by jgaylor** [comment]
@holmesworcester I've updated this to reflect the changes on mobile:
1. Removed 'Drafts' and the related tab setup (drafts | scheduled) from mobile only.
2. Should we keep the desktop setup with the tabs for drafts and scheduled, or do you have something else in mind? Might need to chat about this one.
3. Here's a [loom](https://www.loom.com/share/db04c8ff6bb447adb0e5be32ddac4958) explaining, and the [Figma](https://www.figma.com/file/WcMFt8ntHvNXammfEosbbZ/Scheduled-send?type=design&node-id=871-7996&mode=design&t=oHKSLmRPoMM8964L-4)

**Comment by holmesworcester** [comment]
I think the existing design on desktop is still good. We might want to revisit threads.
